### ISSUE TITLE
a11y: Embedded menu for links are not accessible via keyboard

### ISSUE THREAD

**Issue by rewodev** [quilljs/quill#4205]
The menu items of the embedded menu when selecting the link in the editor are not accessible via keyboard.

**Steps for Reproduction**

1. Visit [Quill Playground](https://quilljs.com/playground/snow#codeN4Ig9AlgdgJgpgDwHQAsAuBbANiAXCAHhggDcACCGAXgB0Q5i0B7AJzoD4CxiT2ap+BAM4BjFhAAOaMkJYja4aPGQArIRy6jxU9iAA0i2IiRq8IEUyhDpARwCuELFjJUyUOAHcyARQdOAFADkAMQMEMwsgXpkwPxkZBhMMHZYcEK4MXHxZMxMWABGAIYsGQDaWdlkpcBkKHCF8CVVAIzRAEzRAGaFWEJwALpkAL79ehXZpYH5eTBRZIHhPRAic4F2RixY0HCBo+PxkxAYhQDmO9GBFvAAtPlYTCIA1qtbUM97UJUf8UNjn2QSLCFERwFAzOBNQIAYSYGAkTD6ZEKnzgEmWSAxUSyaDqGDgGUCQigTA8czAYDIrHm+Ts+TuO34QwAlABuEBDIA)
2. Select link in editor
3. Try to step into embedded menu with keyboard

**Expected behavior**:

The embedded menu is reachable via keyboard.

**Actual behavior**:

The embedded menu is not reachable via keyboard.
